cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -84,6 +85,69 @@ Instead they are defined in terms of their underlying _privileges_ and they can
 In addition, any new roles created can by assigned any combination of _privileges_ to create the specific access control desired.
 A major additional capability is _sub-graph_ access control whereby read-access to the graph can be limited to specific combinations of label, relationship-type and property.
 
+[[administration-security-syntax-rules]]
+== Syntax summaries
+Almost all administration commands have variations in the commands. Parts of the command that are optional or can have multiple values are most common.
+To show all versions of a command, a summary of the syntax will be presented. These summaries will use some special characters to indicate such variations.
+
+The special characters and their meaning are as follows:
+
+.Special characters in syntax summeries
+[options="header", width="100%", cols="1a,3,3"]
+|===
+| Character | Meaning | Example
+
+| `\|`
+| Or, used to indicate alternative parts of a command. Needs to be part of a grouping.
+| If the syntax needs to specify either a name or `*`, this can be indicated with `name \| *`.
```

Comment:
```suggestion
| If the syntax needs to specify either a name or `*`, this can be indicated with `* \| name`.
```
To make the order more consistent (have a card to update other places)